www/js/services/shareLocalDBFile.ts

```diff
@@ -0,0 +1,116 @@
+import i18next from 'i18next';
+import { displayError, displayErrorMsg, logDebug, logWarn } from '../plugin/logger';
+
+function localDBHelpers(fileName: string, fileExtension: string = '.txt') {
+  async function localCopyFile() {
+    return new Promise<void>((resolve, reject) => {
+      let pathToFile, parentDirectory;
+      if (window['cordova'].platformId == 'android') {
+        parentDirectory = window['cordova'].file.dataDirectory.replace('files', 'databases');
```

Comment:
Can you please add in examples of what this manipulation looks like, either as comments here, or in a comment in the PR, which you add a link as a comment here? I see that @the-bay-kay had several comments around why this wasn't working and I want to make sure that future developers are able to see what is supposed to happen.